### PR TITLE
Documentation Update

### PR DESCRIPTION
**Pull request by swatisehgal**
- Update the documentation to capture the deployment of Topology aware scheduling on Kubernetes clusters 1.23  or higher.
- Capture the contributors and maintainers
- Capture links to agenda doc, slack channel and video recordings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,3 +31,20 @@ We *do not* plan to keep permanent forks or to maintain components under this or
 ## what does this repository contain?
 
 This repository contain generic documentation which doesn't belong to specific repositories, maybe because, like this README, covers the organization, or because covers general topics.
+
+## Community, discussion, contribution, and support
+
+You can reach the maintainers of this project at:
```
Comment (ffromani) [inline README.md:39]: I'd add a note that this is a working group in the general, english sense of the word "A working group, or working party, is a group of experts working together to achieve specified goals. " (https://en.wikipedia.org/wiki/Working_group) and not a official k8s working group. While we want to cooperate with k8s, we don't have official endorsement, acknowledgement or anything else

**Review by ffromani — COMMENTED** [review]
/lgtm
with the exception of a possible note that can however be added later
feel free to merge, or please ping me if you want to re-upload

File: README.md (hunk 1 above)
Comment (swatisehgal) [reply README.md:39]: Done!

**Review by ffromani — COMMENTED** [review]
/lgtm

**Pull request merged**